stratosphere/pm/source/pm_main.cpp

```diff
@@ -69,30 +69,35 @@ void __libnx_initheap(void) {
     fake_heap_end   = (char*)addr + size;
 }
 
-void RegisterPrivilegedProcessesWithFs() {
-    /* Ensures that all privileged processes are registered with full FS permissions. */
-    constexpr u64 PRIVILEGED_PROCESS_MIN = 0;
-    constexpr u64 PRIVILEGED_PROCESS_MAX = 0x4F;
-
-    const u32 PRIVILEGED_FAH[0x1C/sizeof(u32)] = {0x00000001, 0x00000000, 0x80000000, 0x0000001C, 0x00000000, 0x0000001C, 0x00000000};
-    const u32 PRIVILEGED_FAC[0x2C/sizeof(u32)] = {0x00000001, 0x00000000, 0x80000000, 0x00000000, 0x00000000, 0xFFFFFFFF, 0xFFFFFFFF, 0x00000000, 0x00000000, 0xFFFFFFFF, 0xFFFFFFFF};
-
-    u32 num_pids;
-    u64 pids[PRIVILEGED_PROCESS_MAX+1];
-    if (R_SUCCEEDED(svcGetProcessList(&num_pids, pids, sizeof(pids)/sizeof(pids[0])))) {
-        for (u32 i = 0; i < num_pids; i++) {
-            const u64 pid = pids[i];
-            if (PRIVILEGED_PROCESS_MIN <= pid && pid <= PRIVILEGED_PROCESS_MAX) {
-                fsprUnregisterProgram(pid);
-                fsprRegisterProgram(pid, pid, FsStorageId_NandSystem,  PRIVILEGED_FAH, sizeof(PRIVILEGED_FAH), PRIVILEGED_FAC, sizeof(PRIVILEGED_FAC));
+namespace {
+
+    static constexpr u32 PrivilegedFileAccessHeader[0x1C / sizeof(u32)]  = {0x00000001, 0x00000000, 0x80000000, 0x0000001C, 0x00000000, 0x0000001C, 0x00000000};
```

Comment:
static is redundant with the anonymous namespace.

Reply:
Right.